airflow/providers/telegram/operators/telegram.py

```diff
@@ -61,6 +61,7 @@ def __init__(
         **kwargs,
     ):
         self.chat_id = chat_id
+        self.text = text
```

Comment:
Why was this problem not detected by unit tests?

Reply:
Ahh.... I know what happened. You should make a slight different change.
```
-    template_fields = ('text', 'chat_id')
+    template_fields = ('telegram_kwargs', 'chat_id')
```
or move the following part of code to execute method:
```
        if text is not None:
            self.telegram_kwargs['text'] = text
```